### PR TITLE
Ability create Profile with empty fields. Add state to Profile model.

### PR DESCRIPTION
**Pull request by mnaichuk**

### CODE REVIEW

File: app/models/profile.rb [hunk 1]
```diff
@@ -101,4 +112,23 @@ def squish_spaces
...
+  def update_profile_label
+    profile_label = user.labels.find_by(key: :profile)
+    return unless profile_full? && profile_label.present?
+
+    profile_label.update(value: :verified)
+  end
+
+  def create_profile_label
+    if profile_full?
+      user.labels.create(key: :profile, value: :verified, scope: :private)
```
Comment (ec) [inline app/models/profile.rb]: user.add_level_label('profile')
user.add_level_label('profile', 'partial')

File: db/migrate/20190902032709_add_state_profiles.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddStateProfiles < ActiveRecord::Migration[5.2]
+  def change
+    add_column :profiles, :state, :integer, limit: 1, after: :country
```
Comment (ec) [inline db/migrate/20190902032709_add_state_profiles.rb]: unsigned

File: spec/api/v2/resource/profiles_spec.rb [hunk 3]
```diff
@@ -95,10 +81,98 @@
...
+          post url, params: {}, headers: auth_header
+        end
+
+        subject { Profile.last }
+
+        it { expect(response.status).to eq(201) }
+
+        it { Profile::OPTIONAL_PARAMS.each { |p| expect(JSON.parse(response.body)[p].blank?).to be_truthy } }
+
+        it { expect(JSON.parse(response.body)['state']).to eq('partial') }
```
Comment (ec) [inline spec/api/v2/resource/profiles_spec.rb]: json_body

File: spec/models/profile_spec.rb [hunk 4]
```diff
@@ -21,36 +21,142 @@
...
+
+        it { expect(subject.metadata.nil?).to be_truthy }
+
+        it { expect(subject.state).to eq('completed') }
+
+        it { expect(subject.user.labels.find_by(key: 'profile').value).to eq('verified') }
       end
     end
+
+    context 'all profile params' do
```
Comment (ec) [inline spec/models/profile_spec.rb:127]: move higher in describe

**Review by ec — CHANGES_REQUESTED** [review]

**Review by ec — APPROVED** [review]

**Review by mod — APPROVED** [review]

**Pull request merged**